### PR TITLE
Add a note to the README to get past potential builds issues on MacOS

### PR DESCRIPTION
**Pull request by jaikiran**
Added a brief note to get past build issues on MacOS which are noted in https://github.com/oracle/graal/issues/2017

### CODE REVIEW

File: vm/README.md [hunk 1]
```diff
@@ -43,6 +43,32 @@ In our CI, we build it using:
 - `binutils`: `2.23.2`
 - `llvm`: `3.8` on linux; `4.0.1` on darwin
 
+---
+**NOTE:**
+
+On certain versions of MacOS you might run into build errors of the form:
```
Comment (dougxc) [inline vm/README.md]: Please specify which versions. This seems specific to 10.15 (Catalina).

File: vm/README.md [hunk 2]
```diff
@@ -43,6 +43,32 @@ In our CI, we build it using:
...
+
+```
+'stdlib.h''stdio.h'  filefile  notnot  foundfound
```
Comment (dougxc) [inline vm/README.md]: That looks like interleaved output. Can you please add `--serial` to the `build` command to get non-interleaved output.

File: vm/README.md (hunk 1 above)
Comment (jaikiran) [reply vm/README.md]: @dougxc, thank you for the review. I've updated the PR with the change.

File: vm/README.md (hunk 2 above)
Comment (jaikiran) [reply vm/README.md]: Done. PR updated with more clear error message.

**Comment by fniephaus** [comment]
Does ```SDKROOT=`xcrun --show-sdk-path` mx --env ce build``` work for you?

**Comment by jaikiran** [comment]
> Does SDKROOT=`xcrun --show-sdk-path` mx --env ce build work for you?

@fniephaus, I just gave that a try and this command worked too. Thank you.

**Pull request closed without merging**